buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -268,6 +272,9 @@ void freeChunk(PoolChunk<T> chunk, long handle, int normCapacity, SizeClass size
                 }
             }
             destroyChunk = !chunk.parent.free(chunk, handle, normCapacity, nioBuffer);
+            if (destroyChunk) {
+                ++pooledChunkDeallocations;
```

Comment:
We could add a comment that all other `destroyChunk` calls comes from the arena itself being finalized, and so don't need to be counted.